react-migration-toolkit/src/components/react-bridge.gts

```diff
@@ -61,24 +61,27 @@ export default class ReactBridge<
   R = ComponentType,
 > extends Component<ReactBridgeArgs<T, R>> {
   tagName = (this.args.tagName ?? 'div') as T;
+  or = (a: unknown, b: unknown) => a ?? b;
   <template>
     {{#let (element this.tagName) as |Tag|}}
-      <Tag
-        class={{unless @tagName 'react-bridge-wrapper'}}
-        data-test-react-bridge-component
-        {{! @glint-nocheck }}
-        {{reactModifier
-          reactComponent=@reactComponent
-          props=(transformPropsWithLegacyContent @props)
-          providerOptions=@providerOptions
-          hasBlock=(has-block)
-        }}
-        ...attributes
-      >
-        {{~#if (has-block)~}}
-          {{yield}}
-        {{/if}}
-      </Tag>
+      {{#let (this.or @hasBlock (has-block)) as |normalizedHasBlock|}}
```

Comment:
Looks like a good patch to solve the problem we're facing with wrapping the bridge and yielding content. The original fix (using `has-block`) was already a patch. Long term, I think we'll need to fix the root cause in [the modifier](https://github.com/qonto/react-migration-toolkit/blob/main/react-migration-toolkit/src/modifiers/react-modifier.ts#L73). 

Could you add it as an issue in github so we'll keep track of it?